v63004/u_inref/gtm8922.csh

```diff
@@ -0,0 +1,71 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+echo '# Testing VIEW <KEYWORD>[:<region-list>]'
+
+setenv ydb_poollimit "50"
+setenv gtm_test_jnl SETJNL
+
+echo "# Create a 3 region DB with gbl_dir mumps.gld and regions DEFAULT, AREG, and BREG"
+$gtm_tst/com/dbcreate.csh mumps 3 >>& dbcreate_log.txt
+
+$MUPIP set -access_method=BG -file mumps.dat >>& dbcreate_log.txt
+$MUPIP set -access_method=BG -file a.dat >>& dbcreate_log.txt
+$MUPIP set -access_method=BG -file b.dat >>& dbcreate_log.txt
+
+echo '# Run generalTest to test all commands of the form VIEW <KEYWORD>[:<region-list>]'
+echo '# Tests for:'
+echo '#    	-VIEW commands accepting region sub-argument accept comma (,) delimited region lists'
+echo '# 	-GT.M sorts the regions, eliminating any duplicates from the list. '
+$ydb_dist/mumps -run generalTest^gtm8922
+
+echo '# Run poolLimitTest to test VIEW POOLLIMIT:<REGION>:n[%]'
+echo '# Tests for:'
+echo '#    	-VIEW commands accepting region sub-argument accept comma (,) delimited region lists'
+echo '# 	-GT.M sorts the regions, eliminating any duplicates from the list. '
+echo '#		-If the VIEW argument has a corresponding environment variable to set the default state,'
+echo '# 		 the state applies to databases as the application implicitly opens them with references.'
+$ydb_dist/mumps -run poolLimitTest^gtm8922
+
+echo '# Run gvsResetTest to test VIEW GVSRESET:<REGION>'
+echo '# Tests for:'
+echo '#    	-VIEW commands accepting region sub-argument accept comma (,) delimited region lists'
+echo '# 	-GT.M sorts the regions, eliminating any duplicates from the list. '
+$ydb_dist/mumps -run gvsResetTest^gtm8922
+
+# The openRegionsTest will run a VIEW command using our cmdline args
+# as its VIEW options. So the options var will be a list
+# of every VIEW option to test
+
+set options=\
+("DBFLUSH:AREG,BREG,BREG" "DBFLUSH" "DBSYNC:AREG,BREG,BREG"\
```

Comment:
Can we include a few other permutations of AREG,BREG,BREG i.e. BREG,AREG,BREG, BREG,BREG,AREG etc. That should not affect the reference file output in any ways but ensures that the sorting is done internally as mentioned in the release note.